tests/integration/docker_test/docker-compose.yml

```diff
@@ -14,4 +14,16 @@ services:
       - ../../../mod.ts:/var/www/docker-test/mod.ts
       - ../../../tsconfig.json:/var/www/docker-test/tsconfig.json
     command: bash -c "tail -f /dev/null"
+    working_dir: /var/www/docker-test
+  remotes:
+    container_name: remotes
+    restart: always
+    volumes:
+      - ./custom_scripts:/var/www/docker-test/custom_scripts
+    build:
+      context: .
+      dockerfile: drivers.dockerfile
+    ports:
+      - "9292:9292"
+    command: bash ./custom_scripts/remote_chrome.sh
```

Comment:
mount `.` too, then when running remote tests, exec into the container and run them

Reply:
I kind of get what you mean, but as mentioned above, i need to keep the browser as far from the execution machine as possible, so will try to come up with a docker network to do so and implement what you have suggested

Reply:
agreed, ignore what i said